### PR TITLE
Add the release note for 1.26.2

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

**Comment by minwoox** [comment]
screencapture-localhost-8000-release-notes-1-26-2-2023-11-08-12_18_59

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

File: site/src/pages/release-notes/1.26.2.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+date: 2023-11-08
+---
+
+## 🛠️ Bug fixes
+
+- The same `EventLoop` is used for an <type://Endpoint> within an <type://EndpointGroup> when
+  used in conjunction with a <type://RetryingClient>. This ensures efficient connection reuse. #5290
```
Comment (trustin) [inline site/src/pages/release-notes/1.26.2.mdx:8]: How about this:

```
Armeria now reuses the connections more efficiently when <type://RetryingClient>
is used in conjunction with client-side load-balancing. Previously, an excessive
number of connections were created when requests are retried. #5290
```

File: site/src/pages/release-notes/1.26.2.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- You no longer see an <type://EndpointSelectionTimeoutException> when selecting an <type://Endpoint> from an
+  <type://EndpointGroup>. #5281
+- Fixed a bug where Armeria Spring integration is not compatible with Spring Boot versions 2.6 and earlier. #5279
```
Comment (trustin) [inline site/src/pages/release-notes/1.26.2.mdx:11]:
```suggestion
- Fixed a bug where Armeria Spring Boot integration is not compatible with Spring Boot 2.6 and earlier. #5279
```

File: site/src/pages/release-notes/1.26.2.mdx (hunk 1 above)
Comment (minwoox) [reply site/src/pages/release-notes/1.26.2.mdx]: Applied. 😄

**Review by ikhoon — APPROVED** [review]
🙇‍♂️🙇‍♂️

**Review by trustin — APPROVED** [review]

**Pull request merged**